docs/cookbook/index.rst

```diff
@@ -122,5 +122,6 @@ Frontend
     :hidden:
 
     frontend/admin-js-and-css
+    frontend/google-analytics.rst
```

Comment:
I guess you forgot to add it here https://github.com/Sylius/Sylius/blob/master/docs/cookbook/frontend/map.rst.inc